flux/flux-core/src/test/java/org/apache/storm/flux/multilang/MultilangEnvironmentTest.java

```diff
@@ -38,19 +38,19 @@ public class MultilangEnvironmentTest {
     public void testInvokePython() throws Exception {
         String[] command = new String[]{"python", "--version"};
         int exitVal = invokeCommand(command);
-        assertEquals("Exit value for python is 0.", 0, exitVal);
+        assertEquals(0, exitVal, "Exit value for python is 0.");
     }
 
     @Test
     public void testInvokeNode() throws Exception {
         String[] command = new String[]{"node", "--version"};
         int exitVal = invokeCommand(command);
-        assertEquals("Exit value for node is 0.", 0, exitVal);
+        assertEquals(0, exitVal, "Exit value for node is 0.");
```

Comment:
Error message should say "Exit value for node should be 0" or something similar ("....was not zero")

Reply:
done